postupgrade/utils.js

```diff
@@ -5,15 +5,77 @@ const simpleGit = require('simple-git/promise')();
 
 /**
  * Merges two comment-json files, with the original having higher priority.
- * @param {string} updatedCommentJson 
- * @param {string} originalCommentJson 
+ * Prunes away duplicated commented out props.
+ *
+ * @param {string} newJson 
+ * @param {string} originalJson 
  * @returns {string}
  */
-exports.mergeJson = function(updatedCommentJson, originalCommentJson) {
-  const merged = assign(parse(updatedCommentJson), parse(originalCommentJson));
+exports.mergeGlobalConfigs = function (newJson, originalJson) {
+  const originalParsed = parse(originalJson);
+  const commentsFromOriginal = parseCommentedOutProps(originalParsed);
+  const newPruned = pruneDuplicatedComments(parse(newJson), commentsFromOriginal);
+  const merged = assign(newPruned, originalParsed);
   return stringify(merged, null, 2);
 }
 
+/**
+ * Parses "commented out" config values.
```

Comment:
Does this method get all of the "commented out" config values? If so, I think we should make that a bit clearer in either the function name or JSDocs.